app/src/main/java/com/pb/nex26dv/shippingapp/orderplacing.java

```diff
@@ -0,0 +1,566 @@
+                    @Override
+                    public void onClick(View v) {
+                        JSONString = mapShipmentOnButtonClick();
+                        HitWebservice hitWebservice = new HitWebservice();
+                        hitWebservice.execute();
+
+                        /*Intent intent = new Intent("com.pb.nex26dv.shippingapp.conformationpage");
+                        startActivity(intent);*/
+                    }
+                }
+        );
+    }
+
+
+//    public void OnClickButtonListener_addtocart() {
+//
+////        System.err.println("EVERYTHING IS OK ADD TO CART");
+//
+//
+////        Toast.makeText(orderplacing.this, "Not Available addto cart", Toast.LENGTH_LONG).show();
+//
+////        addtocart = (Button) findViewById(add_cart);
+////        addtocart.setOnClickListener(
+////                new View.OnClickListener() {
+////                    @Override
+////                    public void onClick(View v) {
+////                        Intent intent = new Intent("com.pb.nex26dv.shippingapp.wishlist");
+////                        startActivity(intent);
+////                    }
+////                }
+////        );
+//    }
+
+
+//    public void OnClickButtonListener_cancel_order() {
+//
+////        System.err.println("EVERYTHING IS OK CANCEL");
+//
+////        Toast.makeText(orderplacing.this, "Not Available cancel", Toast.LENGTH_LONG).show();
+//
+////        cancel_order = (Button) findViewById(order_cancel);
+////        cancel_order.setOnClickListener(
+////                new View.OnClickListener() {
+////                    @Override
+////                    public void onClick(View v) {
+////                        Intent intent = new Intent("com.pb.nex26dv.shippingapp.loginActivity");
+////                        startActivity(intent);
+////                    }
+////                }
+////        );
+//    }
+
+
+    /*public void OnClickButtonListener_back_button() {
+
+        back_button = (Button) findViewById(backbutton);
+        back_button.setOnClickListener(
+                new View.OnClickListener() {
+                    @Override
+                    public void onClick(View v) {
+                        Intent intent = new Intent("com.pb.nex26dv.shippingapp.loginActivity");
+                        startActivity(intent);
+                    }
+                }
+        );
+    }*/
+
+
+    @SuppressWarnings("deprecation")
+    private void onSelectFromGalleryResult(Intent data) {
+
+        Bitmap bm = null;
+        if (data != null) {
+            try {
+                bm = MediaStore.Images.Media.getBitmap(getApplicationContext().getContentResolver(), data.getData());
+
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+
+        //ivImage.setImageBitmap(bm);
+    }
+
+    public String mapShipmentOnButtonClick() {
+        SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you don?t call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.